### DISCUSSION TITLE
Albums List - allow Descending sort

### DISCUSSION THREAD

**Discussion by MisterBleys** [ente-io/ente#1462] [Enhancements]
In the UI (both desktop and mobile), on the albums screen you can order by **Name, Newest, Last updated**. 
Particularly for name, it would be nice to order in Descending order as well. 

I name my albums prefixed by Year, and sometimes I want to order by the latest. Technically I should be able to use by Newest or Last Updated but neither is necessarily the latest in chronology.

I know there's a bunch of sorting feature requests already, but possibly the complexity of this wouldn't be too much (but that's a naive assumption on my part :) )

**Comment by vishnukvmd** [comment]
Thanks for the feedback! The complexity will be in offering a simple UX for all these options, will keep this in mind when we iterate on these screens :)
